game_states.py

```diff
@@ -71,6 +131,21 @@ def render(self):
         for paddle in self.game.players.values():
             self.game.screen.blit(paddle.surface, paddle.position)
 
+        # get time we still have to wait and render it to center of screen
+        text = str(self.delay - self._time_elapsed())
+        time_left_surface = PygameSurfaceDecorator(
+            self.font.render(text, False, pymlgame.DARKYELLOW, pymlgame.BLACK)
```

Comment:
Yes `DARKYELLOW` looks better on the real matelight^^ at least that's my impression^^